[AzureMonitorexporter] fix operation name (#41623)
Azure/azure-sdk-for-net: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md

### Bugs Fixed

> * Fix issue that caused Request OperationName to display routing pattern instead of url path.

Reviewer: ```suggestion
* Fix issue that caused Request OperationName to display identical routing pattern for different paths.
```
